java/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -315,6 +315,9 @@ error.user.account.locked.admin.initiated=Your account is locked. Please contact
 application.not.found=Cannot find an application associated with the given consumer key.
 callback.not.match=Registered callback does not match with the provided url.
 invalid.redirect.uri=Redirect URI is not present in the authorization request.
+par.invalid.request.uri=A PAR request does not exist for the given uuid.
```

Comment:
asgardeo references are present in this file, which has been removed y'day with 96f5b20